Tests/scripts/validate.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 set -e
 
+export DEMISTO_README_VALIDATION=yes
```

Comment:
why do we need this here? `CI` is automatically defined in the build:
![image](https://user-images.githubusercontent.com/30797606/87437261-c078f280-c5f6-11ea-982e-c09eb88228d6.png)

Reply:
I thought so too, but the readme was errornous and it passed the build. see in the last pr: https://github.com/demisto/content/pull/7442